iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ShuntCompensator.java

```diff
@@ -45,21 +45,46 @@
     ShuntCompensator setCurrentSectionCount(int currentSectionCount);
 
     /**
-     * Get the susceptance for the current section count.
+     * Get the susceptance (in S) for the current section count i.e. the sum of the sections' susceptances for all sections in service.
+     * @see #getCurrentSectionCount()
      * <p>
      * Depends on the working variant.
      * @see VariantManager
      */
     double getCurrentB();
 
     /**
-     * Get the conductance for the current section count.
+     * Get the conductance (in S) for the current section count i.e. the sum of the sections' conductances for all sections in service.
+     * If a conductance of a section in service is undefined, it is considered equal to 0.
+     * @see #getCurrentSectionCount()
      * <p>
      * Depends on the working variant.
      * @see VariantManager
      */
     double getCurrentG();
 
+    /**
+     * Get the maximum susceptance (in S) i.e. the sum of all sections' susceptances if it is strictly superior to 0, else 0.
```

Comment:
I understand that you choose 0 because we have the possibility to disconnect the shunt to reach this maximum ?

Reply:
Yes !